src/backend/routes/battleRoutes.js

```diff
@@ -30,9 +30,8 @@ router.post('/prepareMatch1v1', [
 // Body: challengerTankIds and personBeingChallengedId
 // Returns the newly created batlleRecordId
 router.post('/prepareMatch3v3', [
-	check('challengerTankIds', 'Need 3 tank Ids')
-		.isArray({ min: 3, max: 3})
-		.isMongoId(),
+	check('challengerTankIds')
```

Comment:
Are we sure that this change is a good idea?

Reply:
Why was the array check removed?

Reply:
oh nvm it's there. the MongoId check is removed or it will conflict with turning in arrays with `null`

Reply:
Though it would be nice to keep some message instead of removing it